model/src/main/kotlin/licenses/License.kt

```diff
@@ -20,6 +20,7 @@
 package com.here.ort.model.licenses
 
 import com.fasterxml.jackson.annotation.JsonInclude
+import com.here.ort.spdx.SpdxExpression
```

Comment:
Nit: Blank line before this one.

Commit message: Typo in "allong".

Reply:
Done.